packages/mars-theme/src/index.js

```diff
@@ -44,7 +45,7 @@ const marsTheme = {
        * Add a processor to `html2react` so it processes the `<img>` tags
        * inside the content HTML. You can add your own processors too
        */
-      processors: [image]
+      processors: [image, script]
```

Comment:
Instead of adding the processor on each theme, I think we can add it by default in `html2react`:
https://github.com/frontity/frontity/blob/dev/packages/html2react/src/index.ts#L11

because we are "fixing something we broke". It should be fixed by default.